README.md

```diff
@@ -1,2 +1,16 @@
-# cyberdisc-bot
-An attempted bot for the CyberDiscovery discord server.
+# Cyber Discovery Discord Bot [![Build Status](https://travis-ci.com/CyberDiscovery/cyberdisc-bot.svg?branch=master)](https://travis-ci.com/CyberDiscovery/cyberdisc-bot) [![Discord](https://discordapp.com/api/guilds/409851296116375565/embed.png)](https://discord.gg/AQPh34Y)
+The bot for the Cyber Discovery Community Discord Server!
+
+## Commands
+### General Commands:
+* **`:help`** - Displays information about the usage and syntax of the commands
+
+### Cyber Security Commands:
+* **`:level 13 1`** - Gets information about a specific CyberStart Game challenge.
+* **`:haveibeenpwned email@exmaple.com`** - Searches haveibeenpwned.com for breached accounts.
+* **`:hasitbeenpwned password`** - Searches pwnedpasswords.com for breached passwords.
+
+### Commands for Fun:
+* **`:lmgtfy [-d][-ie] Stupid Question?`** - Returns a LMGTFY URL for the given question.  Adding `-d` will delete the message that instigated the command and `-ie` will enable the internet explainer feature on lmgtfy.
+* **`:react :question:`** - Reacts to the previous message with the space seperated emojis in the requesting message.
```

Comment:
`:react :question:`?

Reply:
That's how you do discord emojis

Reply:
This isn't discord, and even still it won't show up as :question: because you have it in an in-line code block `:question:`